schema.sql

```diff
@@ -6,3 +6,5 @@ CREATE TABLE animals(
     neutered bit NOT NULL,
     weight_kg decimal(10,2) NOT NULL
 );
+-- Add new column species
+ALTER TABLE animals ADD species VARCHAR 
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give `value` to `ALTER TABLE`'s `VARCHAR` 🙏🏼 

`[HINT]`

```sql
   ALTER TABLE animals ADD species varchar(255);
```

Reply:
Thanks